src/OptiHPLCHandler/empower_methodset_method.py

```diff
@@ -1,10 +1,11 @@
 import logging
-from typing import Union
+from typing import List, Union
 
 from OptiHPLCHandler.data_types import EmpowerMethodSetMethodModel as DataModel
 from OptiHPLCHandler.empower_instrument_method import (
     ColumnOvenMethod,
-    InstrumentMethod,
+    EmpowerInstrumentMethod,
```

Comment:
I guess you call it EmpowerInstrumentMethod to differentiate from the basis method and the method you will send

Reply:
there is a chance we want to expand it to other chromatographic systems later, and they are bound to work differently, so they would need their own classes.